### PR TITLE
matplotlib 3.3.4

### PR DESCRIPTION
**Pull request by sumanthratna**
partially supersedes #1174 
- [ ] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

### CODE REVIEW

**Comment by sumanthratna** [comment]
@SeekingMeaning ready for review! 
- removes an unnecessary python dep (tabulate) in kat, since Python functionality isn't installed
- bumps matplotlib and removes an unnecessary requirement

**Comment by sjackman** [comment]
Thanks for this work, Sumanth! Could you please squash it down to two commits, one for `matplotlib` and one for `kat`? Happy to explain further if you need more detailed instructions on how to squash. In short, `git rebase -i origin/master` is your friend.

**Comment by sumanthratna** [comment]
> Thanks for this work, Sumanth! Could you please squash it down to two commits, one for `matplotlib` and one for `kat`? Happy to explain further if you need more detailed instructions on how to squash. In short, `git rebase -i origin/master` is your friend.

Sure! I know how to rebase :) one quick question: shouldn't I rebase from the `develop` branch of origin (not `master`)?

**Comment by sjackman** [comment]
>  one quick question: shouldn't I rebase from the develop branch of origin (not master)?

Yep! The two branches should usually be in sync, except immediately after merging a PR before the bottles have been uploaded, but yes, you're right.

File: Formula/matplotlib.rb [hunk 1]
```diff
@@ -1,101 +1,60 @@
...
   homepage "https://matplotlib.org"
-  url "https://files.pythonhosted.org/packages/10/5f/10c310c943f29e67976dcc26dccf9305a5a9bc7483e631ee74a0f95aa5b2/matplotlib-2.2.5.tar.gz"
-  sha256 "a3037a840cd9dfdc2df9fee8af8f76ca82bfab173c0f9468193ca7a89a2b60ea"
-  head "https://github.com/matplotlib/matplotlib.git"
-
-  bottle do
-    root_url "https://linuxbrew.bintray.com/bottles-bio"
-    sha256 "609aaf08da967a0a7729bfb9502f29f9303617154612195f0b2100c4dd2797a7" => :catalina
-    sha256 "d118af398ad5341b4eb16f3e44195ceae12295311a819b6e0c6b14e8558c2c52" => :x86_64_linux
-  end
```
Comment (sjackman) [inline Formula/matplotlib.rb:32]: No need to delete the `bottle do` block. It gets updated automatically by CI. PR looks good otherwise. Thanks!

**Review by sjackman — CHANGES_REQUESTED** [review]

**Pull request merged**